Fails to build on 32-bit x86

Regressed by #1328, see [error log](https://github.com/xiph/rav1e/files/3249965/rav1e-s20190604.log).

```rust
error[E0433]: failed to resolve: use of undeclared type or module `nasm_rs`
  --> build.rs:41:3
   |
41 |   nasm_rs::compile_library_args(
   |   ^^^^^^^ use of undeclared type or module `nasm_rs`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,7 +42,7 @@ bincode = "1.1"
 arrayvec = "0.4.10"
 signal-hook = { version = "0.1.9", optional = true }
 
-[target.'cfg(target_arch = "x86_64")'.build-dependencies]
+[build-dependencies]
 nasm-rs = { git = "https://github.com/tdaede/nasm-rs", branch="msvc-OUT", optional = true }
 
 [target.'cfg(unix)'.build-dependencies]

